src/main/webapp/dashboard-manager.js

```diff
@@ -49,7 +57,6 @@ class DashboardManager {
     } else {
       // Setup data for standard table view
       this.data = this.initData();
-      /* TODO: once the server can send a list of devices, we can get real data here.
       await (fetch('/devices')
```

Comment:
'/devices' is already called in `this.initData()` in the line above. Should the request below be removed?

Reply:
@Mvalverde00